staff/sergio-luz/app-template/news.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>news</title>
+</head>
+<body>
+    <header>
+        <h1>News</h2>
+    </header>
+    <nav class="topnav" id="myTopnav">
+        <a href="home.html" class="active">Home</a>
+        <a href="login.html" class="active">Login</a>
+        <a href="register.html">Register</a>
+        <a href="contact.html">Contact</a>
+        <a href="news.html">News</a>
+        <a href="about.html">About</a>
+        <a href="profile.html">Profile</a>
+    </nav>
+    <section class="row">
+        <section class="column">
+            <form action="post" method="post">
```

Comment:
why a form in news??? this has no sense. at least in news it would be fine to find sections and articles, as a minimum, but not forms...